db/document.go

```diff
@@ -66,18 +77,52 @@ type syncData struct {
 // In its JSON form, the body's properties are at top-level while the syncData is in a special
 // "_sync" property.
 type document struct {
-	syncData
-	body Body
-	ID   string `json:"-"`
-	Cas  uint64
+	syncData        // Sync metadata
+	_body    Body   // Marshalled document body.  Unmarshalled lazily - should be accessed using Body()
```

Comment:
This was originally just `body`, and I switched it to `_body` during development to make sure I didn't miss any external usages (since we want all callers to use Body() now).

I ended up leaving the underscore there as a reminder to ourselves that this is intended to be an internal-only property (even for callers in the db package).  I don't know if there's an idiomatic way to do that (maybe the idiomatic approach is to move document.go to a 'document' package, so that it's actually private).